lightwood/mixers/nn.py

```diff
@@ -9,6 +9,7 @@
 from torch.cuda.amp import GradScaler
 
 from lightwood.logger import log
+from lightwood.encoders import *
```

Comment:
might want to specify which modules/things are you importing for consistency's sake and clarity

Reply:
to also add, do you need to import all of it?

Reply:
Fair enough, I do not. Only `NumericEncoder` is needed right now. Will fix.